site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -43,12 +43,18 @@ The stock `.env` file will contain the following values, you will want to modify
 ```
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
+# Prior to version 1.19.0, using DATABASE_USER
 DATABASE_USER=fusionauth
```

Comment:
We could sync this up with the variable name. `DATABASE_USERNAME` and then use `${DATABASE_USERNAME}` below. 

Same with `POSTGRES_USERNAME`

Reply:
Made all the changes other than for `POSTGRES_USERNAME`, since `POSTGRES_USER` is defined by the postgres docker image: https://hub.docker.com/_/postgres